src/patternfly/components/Tabs/docs/tabs-nav.md

```diff
@@ -8,3 +8,9 @@ Whenever a list of tabs is unique on the current page, it can be used in a `<nav
 | -- | -- | -- |
 | `aria-label="Local"` | `nav.pf-c-tabs` | Gives the `<nav>` element an accessible name. **Required when `.pf-c-tabs` is used with `<nav>`**
 | `aria-label="Local secondary"` | `nav.pf-c-tabs.pf-m-tabs-secondary` | Gives the `<nav>` element an accessible name. **Required when `.pf-c-tabs.pf-m-tabs-secondary` is used with `<nav>`**
+
+### Usage
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-tabs__button` | `<a>` | Creates a tab link. **Required** |
```

Comment:
Since this class can be used on either `<button>` or `<a>`, it seems like we should mention `<button>` here.

I'm just now noticing we don't have any usage documentaton on Tabs. Should we create an issue to add all the other classes here, too?

Reply:
Hmm... do you see the docs for the other sections? I was just commenting on the deviation here. The docs for the rest of the elements Or can/should you use `<button>` with `<nav>`, too? 

I just noticed we have `.pf-c-tabs | <div>, <nav>` in the section above. Do you think I should pull `<nav>` out of that, and move it down to the nav example? Or leave `<nav>` and add `<a>` to the elements `.pf-c-tabs__button` applies to in the first example's docs?

Reply:
Doh, I completely somehow missed the other doc sections. 😞 

It might be possible that consumers would need to use `<button>` due to not having urls defined for each tab, but want the benefits of using `<nav>`. So I think we'd still list `<button>` here, too.

Reply:
What do you think about `.pf-c-tabs | <div>, <nav>` in the first section? Should I remove `<nav>` so that's only in the nav example's docs?

Reply:
I think it's fine as it is. On one hand, I don't think we need to show every possible html element variation that's possible, and can have the documentation mention elements that could be used but not used in an example. 

But for the `<a>`/`<button>` row for the `<nav>` example, I'm inclined to say that the `<a>` is only relevant to the `<nav>` use case, and therefore, it makes sense to call that out with this example specifically, and not in the global documentation.